### DISCUSSION TITLE
tjsburgersevington.co.uk: nuisance

### DISCUSSION THREAD

**Discussion by matthew-horton** [uBlockOrigin/uAssets#16396] [Q&A]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://tjsburgersevington.co.uk/order-now`

### Category

nuisance

### Description

when i try to block an element, if only selects a fullscreen jpg to block, not the actually element i want to hide.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.46.0
Firefox: 108
filterset (summary): 
  network: 92109
  cosmetic: 128724
  scriptlet: 18661
  html: 866
listset (total-discarded, last-updated): 
  added: 
    https://www.i-dont-care-about-cookies.eu/abp/: 25703-4, 3d.3h.33m
    fanboy-annoyance: 83524-9535, 3d.3h.34m
  default: 
    user-filters: 446-7, never
    ublock-filters: 32446-52, 3d.3h.31m
    ublock-badware: 4298-0, 3d.3h.30m
    ublock-privacy: 304-0, 3d.3h.40m
    ublock-abuse: 77-0, 3d.3h.39m
    ublock-unbreak: 1933-1, 3d.3h.32m
    easylist: 62107-633, 3d.3h.35m
    easyprivacy: 30934-1742, 3d.3h.36m
    urlhaus-1: 6589-0, 22h.49m
    plowe-0: 3666-2, 3d.3h.38m
    ublock-quick-fixes: 478-0, 22h.50m
filterset (user): [array of 446 redacted]
trustedset: 
  added: [array of 7 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 766 ms (selfie)
  maxAssetCacheWait: 341 ms
popupPanel: 
  blocked: 3
  blockedDetails: 
    touch2success.com: 1
    webgains.io: 2
```
</details>

**Comment by stephenhawk8054** [comment]
Use the Element picker (the tooltip icon), then use the slider or choose the element you want to hide in the modal dialog box at your screen's bottom right:

https://github.com/gorhill/uBlock/wiki/Element-picker#once-in-element-picker-mode

**Reply by matthew-horton** [reply]
That works. Thank you.
